Doc/c-api/method.rst

```diff
@@ -7,9 +7,9 @@ Instance Method Objects
 
 .. index:: object: instancemethod
 
-An instance method is a wrapper for a :c:data:`PyCFunction` and the new way
-to bind a :c:data:`PyCFunction` to a class object. It replaces the former call
-``PyMethod_New(func, NULL, class)``.
+.. deprecated:: 3.8
+    The ``instancemethod`` class is deprecated.
```

Comment:
You need to use an indentation of 3 spaces (https://devguide.python.org/documenting/#use-of-whitespace).